scripts/cypress/src/base.config.ts

```diff
@@ -56,6 +56,7 @@ const cypressWebpackConfig = (): Configuration => {
 
 type BaseConfig = Omit<Cypress.ConfigOptions, 'component'> & {
   component: Omit<Cypress.ConfigOptions['component'], 'devServer'> & {
+    specPattern?: string[];
```

Comment:
is this missing from cypress official types ?

Reply:
No, but for some reason, due to the recreation of the `component` type, it was failing. So I added here.

Reply:
I just realized we re-create the `component` property on our own that's why its not there